MAUI/Kanban-Board/Events.md

```diff
@@ -9,6 +9,122 @@ documentation: ug
 
 # Events in .NET MAUI Kanban Board (SfKanban)
 
+## CardTapped
+
+The `CardTapped` event occurs when the card is tapped. We can get the following details from the `CardTapped` event.
+
+* `Column` - Returns the instance of the column containing the tapped card.
+* `Index` - Returns the index position of the tapped card within its column.
+* `Data` - Returns the data of the card that was tapped.
+
+{% tabs %}
+{% highlight XAML hl_lines="3" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding Cards}"
+                 CardTapped="OnKanbanCardTapped">
+    <kanban:SfKanban.BindingContext>
+        <local:KanbanViewModel/>
+    </kanban:SfKanban.BindingContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="2" %}
+
+this.kanban.ItemsSource = new KanbanViewModel().Cards;
+this.kanban.CardTapped += this.OnKanbanCardTapped;
+
+private void OnKanbanCardTapped(object sender, Syncfusion.Maui.Kanban.KanbanCardTappedEventArgs e)
+{
+    var selectedColumn = e.Column;
+    var selectedCardIndex = e.Index;
+    var selectedCardDetails = e.Data;
+}
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="KanbanViewModel.cs" %}
+
+public class KanbanViewModel
+{
+    #region Properties
+
+    /// <summary>
+    /// Gets or sets the collection of <see cref="KanbanModel"/> objects.
+    /// </summary>
+    public ObservableCollection<KanbanModel> Cards { get; set; }
+
+    #endregion
+
+    #region Constructor
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="KanbanViewModel"/> class.
+    /// </summary>
+    public KanbanViewModel()
+    {
+        this.Cards = new ObservableCollection<KanbanModel>();
```

Comment:
@Kerubananthan3761 , add private methods and get card collection in separate method

Reply:
Added private method to get card collection.